src/View/Helper/FormHelper.php

```diff
@@ -135,6 +168,7 @@ class FormHelper extends Helper
      */
     protected $_widgets = [
         'button' => 'BootstrapUI\View\Widget\ButtonWidget',
+        'datetime' => ['BootstrapUI\View\Widget\DateTimeWidget', 'select'],
```

Comment:
`DateTimeWidget` doesn't use `SelectBoxWidget` anymore, so this should be just `'BootstrapUI\View\Widget\DateTimeWidget'`.

Reply:
updated